### DISCUSSION TITLE
How to disable built-in on type format?

### DISCUSSION THREAD

**Discussion by ODtian** [clangd/clangd#1988] [Q&A]
I'm using clangd as vscode lsp and use another extension to format my code since it doesn't support passing default format as string. However clangd still format my code after enter an "enter" (onTypeFormat) which break my default format and wrap indent set in vscode. How to disable this feature?

**Comment by HighCommander4** [comment]
Try `"editor.formatOnType": false` in vscode's User or Workspace Settings.
